src/commonmark/en/content/user/using-the-capture-app.md

```diff
@@ -53,6 +56,60 @@
 > achived by clicking the **Switch to row view** button on the top right of the data entry form. If you are currently in **row view** you
 > can switch to the default form view by clicking the **Switch to form view** button on the top right of the data entry form.
 
+## Adding a relationship
+
+<!--DHIS2-SECTION-ID:capture_add_relationship-->
+
+Relationships can be added either during registration, editing or viewing of an event.
+Currently the **Capture App** only supports *Event to Tracked Entity Instance* relationships. 
+
+1. While in an event, click **Add relationship**.
+
+2. Select the relationship type you want to create.
+
+- You now have to options: **Link to an existing Tracked Entity Instance** or **Create new Tracked Entity Instance**.
+
+### Link to an existing Tracked Entity Instance
+
+3. Click **Link to an existing Tracked Entity Instance**.
+
+- You should now be presented with some options for searching for a **Tracked Entity Instance**.
+  You have the option to select a **program**. If a **program** is selected the attributes are derived from the selected **program**.
+  If no **program** is selected, only the attributes that belong to the **Tracked Entity Instance** will be visible.
+
+    - If the **Tracked Entity Instance** or **program** is configured with a unique attribute, this attribure can be
+      used for finding a spesific **Tracked Entity Instance** or **program**. This attribute should be presented alone.
+      When the unique attribute field has been filled out, click the **Search** button located right below
+      the unique attribute field.
+
+    - If the **Tracked Entity Instance** or **program** has attibutes these can be used for searching by expanding the **Search by attributes** box.
+      When all desired attribute fields have been filled out, click the **Search by attributes** button located at the bottom. You can also limit the search by setting the **Organisation unit scope**. If set to *All accessible* you will search for the **Tracked Entity Instance** in all organisation units you have access to. If you select *Selected*, you will be asked to select which organisation units to search within.
+
+4. After a successful search you will be presented with a list of **Tracked Entity Instances** matching the search criteria.
+   To create a relationship click the **Link** button on the **Tracked Entity Instance** you would like to create a relationship to.
+   
+- If you did not find the **Tracked Entity Instance** you were looking for, you can either click the **New search** or **Edit search** buttons.
+  **New search** will take you to new blank search while **Edit search** will take you back to the search you just did keeping the search criterias.
```

Comment:
criterias => criteria
(I would also prefer "search you just performed" instead of "search you just did")

Reply:
Fixed